tools/dependencies/curl.sh

```diff
@@ -38,6 +38,7 @@ if [[ ! -f $DEPS_PATH/lib/libcurl.a ]]; then
                 --without-zsh-functions-dir \
                 --without-librtmp \
                 --without-libssh2 \
+                --without-libidn2 \
```

Comment:
This line is to avoid the problem of missing idn2 library if your xcode version is 9.2 or above